ckanext/datagovtheme/templates/package/read.html

```diff
@@ -276,6 +276,14 @@ <h3>{{ _('Metadata Source') }}</h3>
     {% endif %}
   {% endif %}
 
+  {% set geoplatform_link = h.get_geoplatform_link(pkg_dict["name"]) %}
+    {% if geoplatform_link %}
+      <section id="dataset-other-resources" class="resources module-content">
+          <h3>Other Data Resources</h3>
+              <a href="{{ geoplatform_link }}">View this on Geoplatform</a>
```

Comment:
Maybe open the external link on the new tab instead of letting user navigating away from catalog?

```suggestion
              <a href="{{ geoplatform_link }}" target="_blank">View this on Geoplatform</a>
              <svg class="usa-icon" aria-hidden="true" role="img">
                  <use xlink:href="#svg-launch"></use>
                  <symbol viewBox="0 0 24 24" id="svg-launch" xmlns="http://www.w3.org/2000/svg"><path d="M19 19H5V5h7V3H5a2 2 0 0 0-2 2v14a2 2 0 0 0 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"></path></symbol>
              </svg>
```

Reply:
yeah i like that. good idea